### PR TITLE
Task 76 default mosaic namespace

### PR DESCRIPTION
**Pull request by NemTechCN**
update default mosaic namespace, get them from genesis block

### CODE REVIEW

File: src/common/vue/menu-bar/MenuBarTs.ts [hunk 1]
```diff
@@ -272,16 +269,64 @@ export class MenuBarTs extends Vue {
...
             that.isNodeHealthy = false
+            console.log(e)
+        })
+    }
+
+    // get current network mosaic hex by Genesis Block Info
+    getCurrentNetworkMosaic() {
+        const {currentNode} = this
+        const that = this
+        new BlockApiRxjs().getBlockTransactions(currentNode, 1, 100).subscribe((genesisBlockInfoList) => {
```
Comment (decentraliser) [inline src/common/vue/menu-bar/MenuBarTs.ts]: the third getBlockTransactions should be a queryParam

File: src/common/vue/menu-bar/MenuBarTs.ts [hunk 2]
```diff
@@ -272,16 +269,64 @@ export class MenuBarTs extends Vue {
...
+            genesisBlockInfoList.forEach((transactionItem: any) => {
```
Comment (decentraliser) [inline src/common/vue/menu-bar/MenuBarTs.ts]: here the forEach will not stop before it checked all the genesisBlockInfo list

2 options:
1)
const mosaicDefinitionTx = genesisBlockInfoList.find(({ type }) => type ===  TransactionType.MOSAIC_DEFINITION)
const mosaicAliasTx = genesisBlockInfoList.find(({ type }) => type ===  TransactionType.MOSAIC_ALIAS)

2) put a return;

File: src/common/vue/menu-bar/MenuBarTs.ts [hunk 3]
```diff
@@ -272,16 +269,64 @@ export class MenuBarTs extends Vue {
...
+                        let namespaceSortList = []
+                        // get root namespace
+                        namespaceNameResultList.forEach((item, index) => {
+                            if (!item.parentId) {
+                                namespaceNameResultList.splice(index, 1)
+                                namespaceSortList.push(item)
+                            }
+                        })
+                        // namespace sort
+                        for (let i = 0; i < namesapceListLength - 1; i++) {
```
Comment (decentraliser) [inline src/common/vue/menu-bar/MenuBarTs.ts]: typo namesapceListLength
isnt't there a fullName available?

File: src/common/vue/menu-bar/MenuBarTs.ts (hunk 1 above)
Comment (NemTechCN) [reply src/common/vue/menu-bar/MenuBarTs.ts]: fixed

File: src/common/vue/menu-bar/MenuBarTs.ts (hunk 2 above)
Comment (NemTechCN) [reply src/common/vue/menu-bar/MenuBarTs.ts]: fixed

**Pull request merged**